packages/modules/mssqlserver/src/mssqlserver-container.test.ts

```diff
@@ -0,0 +1,81 @@
+import { Wait } from "testcontainers";
+import sql, { config } from "mssql";
+import { MSSQLServerContainer } from "./mssqlserver-container";
+
+describe("MSSqlServerContainer", () => {
+  jest.setTimeout(180_000);
+
+  // connect {
+  it("should connect and return a query result", async () => {
+    const container = await new MSSQLServerContainer()
+      .acceptLicense()
+      .withWaitStrategy(Wait.forLogMessage(/.*Recovery is complete.*/, 1))
```

Comment:
Could this wait strategy be configured in the module itself?

Reply:
I left the wait strategy out of the module to allow users to use their own. You can see an example below. I can merge this branch into the PR and update the docs to include this but I wanted to keep the changes to a minimum just in case. 

[https://github.com/wjin17/testcontainers-node/blob/40385d0af17ef2d9af75165ac99da83c96787470/packages/modules/mssqlserver/src/mssqlserver-container.test.ts#L84](https://github.com/wjin17/testcontainers-node/blob/40385d0af17ef2d9af75165ac99da83c96787470/packages/modules/mssqlserver/src/mssqlserver-container.test.ts#L84)

Reply:
@wjin17 take into account that Testcontainers modules are pre-configured, so, it is totally fair to use the wait strategy in the `mssql-server-container.ts`.

Reply:
@eddumelendez Great point!

I've
1. Merged the change from SA to sa
2. Removed password validation
3. Created a default wait strategy* that can be overwritten
4. Added tests for password validation using the native MSSql instance and a test using the Express edition with environment variables

I had to allow the wait strategy regex to be overwritten because the Express edition requires a different wait regex before the db can be queried.

Let me know if I should add any more changes.

Thanks again for your review!